Gemfile

```diff
@@ -5,7 +5,6 @@ gem 'junos-ez-stdlib', git: 'https://github.com/Juniper/ruby-junos-ez-stdlib.git
 
 group :lint do
   gem 'foodcritic', '~> 5.0'
-  gem 'rubocop', '~> 0.34'
```

Comment:
If you switch testing of this cookbook over to ChefDK as we do in our other cookbooks you'll never have to update this file again.

Reply:
Any chance you have a good cookbook as an example to copy 🍝 from?